puppet/modules/quickstack/manifests/pacemaker/cinder.pp

```diff
@@ -9,6 +9,8 @@
   $backend_glusterfs_name = 'glusterfs',
   $backend_iscsi          = false,
   $backend_iscsi_name     = 'iscsi',
+  $backend_netapp         = false,
```

Comment:
Thanks for clarifying.  Does this look right? https://github.com/rhefner1/astapor/commit/755848df4e27bcca311aa55da2298d0c6e04ca0f